ddl/ddl_db_test.go

```diff
@@ -1155,6 +1158,18 @@ func (s *testDBSuite) TestChangeColumn(c *C) {
 	c.Assert(colC.Comment, Equals, "col c comment")
 	hasNotNull = tmysql.HasNotNullFlag(colC.Flag)
 	c.Assert(hasNotNull, IsFalse)
+
+	s.mustExec(c, "create table test_on_update_c (c1 int, c2 int);")
+	s.mustExec(c, "alter table test_on_update_c add column c3 timestamp null default '2017-02-11' on update current_timestamp;")
```

Comment:
Why put this test here? I think this case checks the operation of `add column`, but `TestChangeColumn` checks the operation of `change column`.